benchmark/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 file(GLOB BENCHMARK "*.cpp")
-add_executable( benchmark ${BENCHMARK} )
+add_executable( benchmark ${BENCHMARK} bn_256.cpp)
```

Comment:
Unneeded -- `BENCHMARK` is a glob of all cpp files

Reply:
resolve with https://github.com/AntelopeIO/leap/pull/280/commits/f1f96883e1bae16a13161a456f201695e91f6498